e2e/README.md

```diff
@@ -196,6 +196,16 @@ class ExampleInput extends React.Component {
 
 It is recommended to follow the scheme parentID/ChildDescription.
 
+## Setting remote config defaults for e2e tests
+
+We do not read remote config values from firebase for e2e tests, so default remote config values are used. A custom set of remote config
+defaults may be needed for some tests which differs from production configuration. For example, in production code,
```

Comment:
'which differs from production configuration' to 'which differ from the production configuration'

Reply:
actually mine has the correct grammar

Reply:
"set" is the subject, which is singular